Hunting Queries/Syslog/SCXRunAsProviderExecuteShellCommand.yml

```diff
@@ -0,0 +1,41 @@
+id: 0d298a1d-1a08-4f4b-8b28-687bfe0012e8
+name: SCX RunAsProvider ExecuteShellCommand
+description: |
+  'This hunting query uses Auditd security events collected via the Syslog data connector to explore the use of the SCX RunAsProvider Invoke_ExecuteShellCommand to execute any UNIX/Linux command using the /bin/sh shell.
+  SCXcore, started as the Microsoft Operations Manager UNIX/Linux Agent, is now used in a host of products including Microsoft Operations Manager. Microsoft Azure, and Microsoft Operations Management Suite. 
+  '
+severity: High
+requiredDataConnectors:
+  - connectorId: Syslog
+    dataTypes: 
+      - Syslog
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+  - Execution
+relevantTechniques:
+  - T1190
+  - T1203
+tags:
+  - SimuLand
+query: |
+  Syslog
+  | parse SyslogMessage with "type=" EventType " audit(" * "): " EventData
+  | project TimeGenerated, EventType, Computer, EventData 
+  // Extract AUOMS_EXECVE details from EventData
+  | where EventType =~ "AUOMS_EXECVE"
+  | parse EventData with * "syscall=" syscall " syscall_r=" * " success=" success " exit=" exit " a0" * " ppid=" ppid " pid=" pid " audit_user=" audit_user " auid=" auid " user=" user " uid=" uid " group=" group " gid=" gid "effective_user=" effective_user " euid=" euid " set_user=" set_user " suid=" suid " filesystem_user=" filesystem_user " fsuid=" fsuid " effective_group=" effective_group " egid=" egid " set_group=" set_group " sgid=" sgid " filesystem_group=" filesystem_group " fsgid=" fsgid " tty=" tty " ses=" ses " comm=\"" comm "\" exe=\"" exe "\"" * "cwd=\"" cwd "\"" * "name=\"" name "\"" * "cmdline=\"" cmdline "\" containerid=" containerid
```

Comment:
I see you have multiple user fields (user, audit_user,effected user...).  Does it make sense to map them all to the Account entity at the end?  Do these end up being the same account in this case?  If not, you can have multiple columnames for the same field mapping.  entityType: Account, identifier: FullName (or whatever proper identifier if not FullName) and columnName: User and then another set with the same identifer but columnName: audit_user

Reply:
done. I used the `user` field for that. It will be root always.